sdk/storage/storage-blob/src/BlobQueryResponse.browser.ts

```diff
@@ -0,0 +1,499 @@
+   */
+  public get blobContentMD5(): Uint8Array | undefined {
+    return this.originalResponse.blobContentMD5;
+  }
+
+  /**
+   * Returns the date and time the file was last
+   * modified. Any operation that modifies the file or its properties updates
+   * the last modified time.
+   *
+   * @readonly
+   * @type {(Date | undefined)}
+   * @memberof BlobQueryResponse
+   */
+  public get lastModified(): Date | undefined {
+    return this.originalResponse.lastModified;
+  }
+
+  /**
+   * A name-value pair
+   * to associate with a file storage object.
+   *
+   * @readonly
+   * @type {(Metadata | undefined)}
+   * @memberof BlobQueryResponse
+   */
+  public get metadata(): Metadata | undefined {
+    return this.originalResponse.metadata;
+  }
+
+  /**
+   * This header uniquely identifies the request
+   * that was made and can be used for troubleshooting the request.
+   *
+   * @readonly
+   * @type {(string | undefined)}
+   * @memberof BlobQueryResponse
+   */
+  public get requestId(): string | undefined {
+    return this.originalResponse.requestId;
+  }
+
+  /**
+   * If a client request id header is sent in the request, this header will be present in the
+   * response with the same value.
+   *
+   * @readonly
+   * @type {(string | undefined)}
+   * @memberof BlobQueryResponse
+   */
+  public get clientRequestId(): string | undefined {
+    return this.originalResponse.clientRequestId;
+  }
+
+  /**
+   * Indicates the version of the File service used
+   * to execute the request.
+   *
+   * @readonly
+   * @type {(string | undefined)}
+   * @memberof BlobQueryResponse
+   */
+  public get version(): string | undefined {
+    return this.originalResponse.version;
+  }
+
+  /**
+   * The SHA-256 hash of the encryption key used to encrypt the blob. This value is only returned
+   * when the blob was encrypted with a customer-provided key.
+   *
+   * @readonly
+   * @type {(string | undefined)}
+   * @memberof BlobQueryResponse
+   */
+  public get encryptionKeySha256(): string | undefined {
+    return this.originalResponse.encryptionKeySha256;
+  }
+
+  /**
+   * If the request is to read a specified range and the x-ms-range-get-content-crc64 is set to
+   * true, then the request returns a crc64 for the range, as long as the range size is less than
+   * or equal to 4 MB. If both x-ms-range-get-content-crc64 & x-ms-range-get-content-md5 is
+   * specified in the same request, it will fail with 400(Bad Request)
+   *
+   * @type {(Uint8Array | undefined)}
+   * @memberof BlobQueryResponse
+   */
+  public get contentCrc64(): Uint8Array | undefined {
+    return this.originalResponse.contentCrc64;
+  }
+
+  /**
+   * The response body as a browser Blob.
+   * Always undefined in node.js.
+   *
+   * @readonly
+   * @type {(Promise<Blob> | undefined)}
+   * @memberof BlobQueryResponse
+   */
+  public get blobBody(): Promise<Blob> | undefined {
+    throw Error(`Quick query in browser is not supported yet.`);
```

Comment:
Is it because avro parser currently only support node.js?

Reply:
Yes.